core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -75,6 +75,16 @@ private[op] class OpMultiClassificationEvaluator
 
   def setTopNs(v: Array[Int]): this.type = set(topNs, v)
 
+  final val topKs = new IntArrayParam(
+    parent = this,
+    name = "topKs",
+    doc = "sequence of topK values to use for top K metrics",
+    isValid = _.forall(_ > 0)
+  )
+  setDefault(topKs, Array(5, 10, 20, 50, 100, 200))
```

Comment:
Let's keep the max topK in line with the max cardinality we're setting for multiclass classification here (100): https://github.com/salesforce/TransmogrifAI/blob/master/core/src/main/scala/com/salesforce/op/stages/impl/tuning/Splitter.scala#L179